examples/agents_tracing-eval_mlflow/readme.md

```
@@ -41,10 +41,20 @@ Both agents were enhanced with:
 
 1. **Install dependencies**
    ```bash
-   pip install -r requirements.txt
+   # path : /examples/agents_tracing-eval_mlflow
+   uv venv --python 3.12
+   source .venv/bin/activate
+   uv pip install -r requirements.txt
    ```
 
 2. **Start Llama Stack server** on port 8321
+   ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)